CHANGELOG.md

```diff
@@ -2,6 +2,16 @@
 
 All notable changes to this project will be documented in this file.
 
+# [1.4.0](https://github.com/sapphiredev/cli/compare/v1.3.1...v1.4.0) - (2022-09-14)
```

Comment:
Revert this. Changelog is auto-generated.

Reply:
ok

Reply:
done